digdag-core/src/main/java/io/digdag/core/workflow/SlaCalculator.java

```diff
@@ -13,22 +17,53 @@
 import com.google.inject.Inject;
 import io.digdag.client.config.Config;
 import io.digdag.client.config.ConfigException;
+import io.digdag.core.agent.CheckedConfig;
+import io.digdag.core.agent.EditDistance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import static java.util.Locale.ENGLISH;
 
+class UnusedConfigException extends ConfigException
```

Comment:
This exception is defined as package private.
I am not sure how and where this exception is handled.
If it is only processed in same package, it is ok.